charts/opensearch/README.md

```diff
@@ -0,0 +1,156 @@
+# OpenSearch Helm Chart
+
+This Helm chart installs [OpenSearch](https://github.com/opensearch-project/OpenSearch) with configurable TLS, RBAC and much more configuration. This chart caters a number of different use cases and setups.
+
+- [Requirements](#requirements)
+- [Installing the chart](#installing)
+- [Uninstalling the chart](#uninstalling)
+- [Configuration](#configuration)
+
+## Requirements
+
+* Kubernetes >= 1.14
+* Helm >= 2.17.0
+* Recommended that you have at least 8 GiB of memory available for this deployment. You can expect the deployment to fail if, say, you have less than 4 GiB of memory available.
+
+## Installing
+
+To install the chart with the release name `my-release`:
+- Switch to opensearch directly after cloning the repo
+  `❯ cd charts/opensearch`
+- Run `❯ helm package .`
+- Install using Helm 3:
+`❯ helm install my-release opensearch-1.0.0.tgz`
```

Comment:
Why do we ask them to package if they are in the chart dir anyway? They can just `helm install opensearch . ` if they are in the repo.

Reply:
Yes they can. I just wanted opinions, as in ODFE this was the doc. I also don't prefer packaging but nevertheless we will move away from it after we have the releasing in place. They users can directly add the repo and install the charts from the hosted repository.